### DISCUSSION TITLE
Seemlessly embed all stories?

### DISCUSSION THREAD

**Discussion by tomdavidson** [storybookjs/storybook#25141] [Help]
### Summary

I would like to embed all the UI components in a different web app for - just for presentation; test and dev goes through SB directly.   Does anyone have experience with  doing this other than one story at  a time?

**Comment by chantastic** [comment]
i believe i'm missing something with this question.
they're just components, right? so, you should be able to render them anywhere that supports the component runtime.
what am i missing?

**Reply by rwcorbett** [reply]
I realize this is an older thread - but is it possible to see an example of how you purpose to render these?

(follow up to question here: in https://github.com/storybookjs/storybook/discussions/24694#discussioncomment-9203634)
